edk2toolext/edk2_invocable.py

```diff
@@ -380,8 +382,16 @@ def ParseCommandLineOptions(self):
             sys.exit(1)
 
         except (FileNotFoundError):
-            # Gracefully exit if we can't find the file
-            print(f"We weren't able to find {settingsArg.platform_module}")
+            if settingsArg.help:
+                try:
+                    print("WARNING: Some command line arguments may be missing. Provide a platform module file to "
```

Comment:
Very minor, but for someone frustrated and quickly trying to figure out what to do, writing "platform module file" in the same way as the parameter name would help them realize it's something they can look below to find and not just a plain English reference to something.

```suggestion
                    print("WARNING: Some command line arguments may be missing. Provide a PLATFORM_MODULE file to "
```